main.js

```diff
@@ -0,0 +1,11 @@
+var $start = document.getElementById('start')
+$start.addEventListener('click', function() {
+  var startWatch = setInterval(addSecond, 1000)
```

Comment:
Why store this variable? It is never used.

Reply:
I assumed we needed to store the variable so that we would know how to reference it later when we needed to clear the interval.

Reply:
Most linters throw an error when you have unused variables. You want to add the code for the current feature, not future features.

Reply:
Got it.  Come to think of it, it doesn't look like my linter is running.  I'm assuming it's because this was started in a new repository.  Let me see if I can figure out how to get it running again.

Reply:
I'm struggling with how to do this without introducing a whole node-modules folder to the repository.